packages/trader/src/AppV2/Components/AccumulatorStats/accumulator-stats.tsx

```diff
@@ -0,0 +1,120 @@
+import React, { useEffect, useMemo, useState } from 'react';
+import { ActionSheet, Text } from '@deriv-com/quill-ui';
+import { LabelPairedChevronDownSmBoldIcon } from '@deriv/quill-icons';
+import { Localize } from '@deriv/translations';
+import { observer } from '@deriv/stores';
+import { useTraderStore } from 'Stores/useTraderStores';
+import StatsRow from './accumulator-stats-row';
+import AccumulatorStatsDescription from './accumulator-stats-description';
+import AccumulatorStatsModal from './accumulator-stats-modal';
+
+const AccumulatorStats = observer(() => {
+    const { ticks_history_stats = {} } = useTraderStore();
+    const [is_open, setIsOpen] = useState(false);
+    const [is_open_description, setIsOpenDescription] = useState(false);
+
+    const ticks_history = useMemo(() => {
+        return ticks_history_stats?.ticks_stayed_in ?? [];
+    }, [ticks_history_stats]);
+
+    const [animationClass, setAnimationClass] = useState('');
+    const [lastValue, setLastValue] = useState<number | null>(null);
+    const [isMovingTransition, setIsMovingTransition] = useState(false);
+
+    const rows: number[][] = useMemo(() => {
+        const row_size = 5;
+        return ticks_history.reduce<number[][]>((acc, _el, index) => {
+            if (index % row_size === 0) {
+                acc.push(ticks_history.slice(index, index + row_size));
+            }
+            return acc;
+        }, []);
+    }, [ticks_history]);
+
+    const onActionSheetClose = () => {
+        setIsOpen(false);
+        setIsOpenDescription(false);
+    };
+
+    useEffect(() => {
+        let successTimeout: NodeJS.Timeout | undefined,
+            errorTimeout: NodeJS.Timeout | undefined,
+            transitionTimeout: NodeJS.Timeout | undefined;
```

Comment:
```suggestion
        let successTimeout: ReturnType<typeof setTimeout>,
            errorTimeout: ReturnType<typeof setTimeout>,
            transitionTimeout: ReturnType<typeof setTimeout>;
```
this way is safer as setTimeout returns a different type depending on if you're running it in Node.js or a browser.